skore-ui/src/stores/project.ts

```diff
@@ -100,7 +100,6 @@ export const useProjectStore = defineStore("project", () => {
   let _stopBackendPolling: Function | null = null;
   async function startBackendPolling() {
     _isCanceledCall = false;
-    await fetch();
```

Comment:
Do we not want to `fetch` as soon as `startBackendPolling` runs?

Reply:
Yes and fetch already does that.
cf https://github.com/probabl-ai/skore/blob/1571501c06652b62bb0067569c02ac28205bf45e/skore-ui/src/services/utils.ts#L90